packages/e2e-tests/specs/editor/various/preferences.test.js

```diff
@@ -46,7 +50,7 @@ describe( 'preferences', () => {
 
 		// Dismiss.
 		await page.click(
-			'.edit-post-sidebar__panel-tabs [aria-label="Close Settings"]'
+			'div[aria-label="Editor settings"] div[role="tablist"] + button[aria-label="Close Settings"]'
```

Comment:
I don't love relying on a sibling selector here. There are technically two `button[aria-label="Close Settings"]`, and being adjacent to the `tablist` is what makes the one we want stand out the most without falling back to a classname. I'm not actually sure this is better, it feels fragile. 🤔

Reply:
The fact that there are two "Close Settings" button should have also been the case before the changes from this PR. It looks like removing the `display: none` rule in `packages/edit-post/src/components/sidebar/style.scss` could be responsible for this test regression.

If re-instatig the `display: none` styles solve the "there are two buttons with the same label" situation, we could either keep the selector unchanged from `trunk` (since the `edit-post-sidebar__panel-tabs` is still being used), or at most change it to

```suggestion
			'div[aria-label="Editor settings"] button[aria-label="Close Settings"]'
```

Reply:
Makes sense! Then I guess we can keep the selector that you wrote. Is the test still failing, though?

Reply:
Nope, this test should be passing just fine